lib/const.rb

```diff
@@ -2,7 +2,11 @@
 
 CREW_VERSION = '1.4.4'
 
-ARCH = `uname -m`.strip
+ARCH_ACTUAL = `uname -m`.strip
+# This helps with virtualized builds on aarch64 machines
+# which report armv8l when linux32 is run.
+ARCH = if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
```

Comment:
After some thought, this does appear to be the correct approach.

Reply:
done.